lib/ansible/modules/system/filesystem.py

```diff
@@ -350,6 +355,7 @@ def main():
             opts=dict(),
             force=dict(type='bool', default=False),
             resizefs=dict(type='bool', default=False),
+            mount_dir=dict(default=False)
```

Comment:
```suggestion
            mount_dir=dict()
```

default=False is default so no need to add.

Reply:
Default value removed.